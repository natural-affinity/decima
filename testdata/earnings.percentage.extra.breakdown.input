decima -b -p 12.5 -e 25 80
