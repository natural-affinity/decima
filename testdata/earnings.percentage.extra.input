decima -p 12.5 -e 10 80
