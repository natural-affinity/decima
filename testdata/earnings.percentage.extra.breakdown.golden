You owe: $35.00

Takehome: $45.00
Earnings: $80.00
 Percent: 12.50%
   Tithe: $10.00
   Extra: $25.00
